Tests/SwiftStdlibTests/MutableCollectionTests.swift

```diff
@@ -33,4 +33,24 @@ final class MutableCollectionTests: XCTestCase {
         array2.sort(by: \String.first, with: optionalCompare)
         XCTAssertEqual(array2, ["Bryant", "James", "Wade", ""])
     }
+    func testAssignToAll() {
+        struct TestStruct: ExpressibleByIntegerLiteral, Equatable {
```

Comment:
Moving this outside should fix the swiftlint warning

> Please make sure to mark the struct as `private` so it is accssesbile from this file only

Reply:
ok, moved it out of the test class, just below

Reply:
I think there is a  file name `TestHelpers` where you can move this class to :)

Reply:
Didn't know about that, thanks
I moved it